aries_vcx/src/handlers/proof_presentation/prover/prover.rs

```diff
@@ -198,7 +199,7 @@ mod tests {
         let setup = SetupLibraryWalletPoolZeroFees::init();
 
         libindy::utils::anoncreds::tests::create_and_store_credential(utils::constants::DEFAULT_SCHEMA_ATTRS, false);
-        let did = setup.faber.config_issuer.institution_did.clone();
+        let did = get_config_value(CONFIG_INSTITUTION_DID).unwrap();
```

Comment:
Just and idea - how about hardcoding the institution did into the tests, or moving it `mockdata/` or `constants.rs`? The reason is I am not enjoying adding another dependency on `settings.rs`

Reply:
I don't object, but alternatively I'd suggest to let `SetupLibraryWalletPoolZeroFees` return the value of institution did

Reply:
OK (I would prefer an object with the value of institution did)